virtual_rainforest/models/animals/animal_cohorts.py

```diff
@@ -303,3 +307,42 @@ def can_reproduce(self) -> bool:
 
         """
         return self.stored_energy >= self.reproduction_energy_threshold
+
+    def is_below_energy_threshold(self) -> bool:
+        """Check if cohort's energy is below a certain threshold.
```

Comment:
Is this only used for migrating cohorts? if so I would be more explicit about it in the docstring